src/Etg.Yams/DIModule.cs

```diff
@@ -22,205 +22,193 @@ namespace Etg.Yams
     /// </summary>
     public static class DiModule
     {
-        public static void RegisterTypes(IUnityContainer container, YamsConfig config)
+        public static void RegisterTypes(ContainerBuilder builder, YamsConfig config)
         {
-            RegisterConfig(container, config);
+            RegisterConfig(builder, config);
 
-            RegisterCloudStorageAccount(container);
+            RegisterCloudStorageAccount(builder);
 
-            RegisterCloudBlobClient(container);
+            RegisterCloudBlobClient(builder);
 
-            RegisterBlobLeaseFactory(container);
+            RegisterBlobLeaseFactory(builder);
 
-            RegisterBlobLeaseFactory(container);
+            RegisterBlobLeaseFactory(builder);
 
-            RegisterProcessFactory(container);
+            RegisterProcessFactory(builder);
 
-            RegisterProcessStopper(container);
+            RegisterProcessStopper(builder);
 
-            RegisterUpdateSessionManager(container);
+            RegisterUpdateSessionManager(builder);
 
-            RegisterApplicationConfigSymbolResolver(container);
+            RegisterApplicationConfigSymbolResolver(builder);
 
-            RegisterApplicationConfigParser(container);
+            RegisterApplicationConfigParser(builder);
 
-            RegisterConfigurableApplicationFactory(container);
+            RegisterConfigurableApplicationFactory(builder);
 
-            RegisterUpdateSessionManagerConfig(container);
+            RegisterUpdateSessionManagerConfig(builder);
 
-            RegisterApplicationDeploymentDirectory(container);
+            RegisterApplicationDeploymentDirectory(builder);
 
-            RegisterApplicationPool(container);
+            RegisterApplicationPool(builder);
 
-            RegisterApplicationInstaller(container);
+            RegisterApplicationInstaller(builder);
 
-            RegisterApplicationDownloader(container);
+            RegisterApplicationDownloader(builder);
 
-            RegisterApplicationUpdateManager(container);
+            RegisterApplicationUpdateManager(builder);
 
-            RegisterDeploymentWatcher(container);
+            RegisterDeploymentWatcher(builder);
 
-			RegisterDeploymentRepository(container);
+            RegisterDeploymentRepository(builder);
         }
 
-        private static void RegisterDeploymentRepository(IUnityContainer container)
+        private static void RegisterDeploymentRepository(ContainerBuilder builder)
         {
-            container.RegisterType<IDeploymentRepository>(new ContainerControlledLifetimeManager(), new InjectionFactory(
```

Comment:
Everywhere you see `ContainerControlledLifetimeManager` means that the class is being registered as a singleton. We need to do the same with Autofac.